dockerfiles/Dockerfile.expo-publisher

```diff
@@ -1,6 +1,6 @@
 FROM node:12-alpine
 
-RUN apk add --update bash git
+RUN apk add --update bash git python
```

Comment:
we need python now?

Reply:
Expo requires it in some modules for publishing.  It seems to find alternatives if it's not available, but they're more flakey than just having it included.